2.0/flask_app/routes/dashboard.py

```
@@ -0,0 +1,3 @@
+from flask import Blueprint, render_template, jsonify
```

Ruff (0.11.9)

`flask.render_template` imported but unused

Remove unused import

(F401)